tasks/exceptions/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+// Rust technically doesn't have exception, but different
+// types of error handling. Here are two examples of results.
+struct WrongStuffHappenedError;
```

Comment:
Do you think we should show implementing the `Error` trait?

Reply:
I've given it a shot, but the source is not much useful here. 
Not sure we should keep it, what do you think?

Reply:
It would be good to use an actual error, I think it's considered bad form to use things that don't `impl Error` to be used in Result like that. :(

Do you think we could use an existing error in the `std` library?

Reply:
Okay, I've changed it now, what to you think? :)